### PR TITLE
Test "add capacity" via UI on tier1 suite

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

**Review by ebenahar — DISMISSED** [review]

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 1]
```diff
@@ -469,41 +471,57 @@ def add_capacity(osd_size_capacity_requested):
...
-        log.info("Patching Local Volume CR...")
-        lvcr.patch(
-            resource_name=lvcr.get()["items"][0]["metadata"]["name"],
-            params=param.strip("\n"),
+    ocp_version = get_ocp_version()
+    platform = config.ENV_DATA.get("platform", "").lower()
+    is_lso = config.DEPLOYMENT.get("local_storage")
+    if (
+        ocp_version == "4.7"
+        and (platform == "aws" or platform == "vsphere")
```
Comment (PrasadDesala) [inline ocs_ci/ocs/resources/storage_cluster.py:479]: Please use constants AWS_PLATFORM and VSPHERE_PLATFORM

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 2]
```diff
@@ -469,41 +471,57 @@ def add_capacity(osd_size_capacity_requested):
...
-        lvcr = localstorage.get_local_volume_cr()
-        log.info("Patching Local Volume CR...")
-        lvcr.patch(
-            resource_name=lvcr.get()["items"][0]["metadata"]["name"],
-            params=param.strip("\n"),
+    ocp_version = get_ocp_version()
+    platform = config.ENV_DATA.get("platform", "").lower()
+    is_lso = config.DEPLOYMENT.get("local_storage")
+    if (
+        ocp_version == "4.7"
```
Comment (PrasadDesala) [inline ocs_ci/ocs/resources/storage_cluster.py:478]: Any reason for running this only on OCP 4.7? Add-capacity UI is same even for older OCP versions and I think also in 4.8

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 3]
```diff
@@ -469,41 +471,57 @@ def add_capacity(osd_size_capacity_requested):
...
-                                                 "value": {final_device_list}}}]"""
-        log.info(f"Final device list : {final_device_list}")
-        lvcr = localstorage.get_local_volume_cr()
-        log.info("Patching Local Volume CR...")
-        lvcr.patch(
-            resource_name=lvcr.get()["items"][0]["metadata"]["name"],
-            params=param.strip("\n"),
+    ocp_version = get_ocp_version()
+    platform = config.ENV_DATA.get("platform", "").lower()
+    is_lso = config.DEPLOYMENT.get("local_storage")
```
Comment (yitzhak12) [inline ocs_ci/ocs/resources/storage_cluster.py:476]: Actually, there is already a function for checking if it's an LSO cluster here: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/cluster.py#L1523. You can use it instead.

File: ocs_ci/ocs/resources/storage_cluster.py (hunk 2 above)
Comment (OdedViner) [reply ocs_ci/ocs/resources/storage_cluster.py:478]: I tested it on OCP4.7.
Maybe on different versions the elements(ui) are different

File: ocs_ci/ocs/resources/storage_cluster.py (hunk 3 above)
Comment (OdedViner) [reply ocs_ci/ocs/resources/storage_cluster.py:476]: I think using this function can cause LOOP
https://github.com/red-hat-storage/ocs-ci/blob/93b1fed9c3ededb20da0278db6efc3e042093c32/ocs_ci/ocs/resources/storage_cluster.py#L71

**Review by vasukulkarni — APPROVED** [review]

File: ocs_ci/ocs/resources/storage_cluster.py (hunk 3 above)
Comment (yitzhak12) [reply ocs_ci/ocs/resources/storage_cluster.py:476]: ok np

**Review by yitzhak12 — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**